src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,11 @@
+package seedu.addressbook.data.person;
+
+/**
+ * A read-only immutable interface for Name, Phone, Email and Address.
```

Comment:
this is not specific to name, phone, email or address, right?
any class that implements this interface can provide the `getPrintableString()` method.
now, in this light, think of a better header description for this one :-)